### PR TITLE
Add html and CSS in the hello world project

### PR DESCRIPTION
**Pull request by KebebewAbabu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,8 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
+#ignored files
+/test.md
```
Comment (sumairq) [inline .gitignore:9]: - [ ] Kindly add the node_modules directory to the `gitignore` file. 
It is always a good idea to add the node_modules directory to the `.gitignore` file in all your projects as it is part of the standard practice.
This way you can avoid putting possibly hundreds of MB of dependencies in your repository. 😄

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @KebebewAbabu , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Linters are passing
✔️ Very Professional Readme
✔️ Correct Github flow

## Required Changes ♻️

- [ ] It would be great to add a nice description of the changes to your Pull Request as it is missing in this case.

_Check the comments under the review._

## Optional suggestions

- [ ] Let's try to use more descriptive names for branches rather than generic names. for example, you can call this branch `hello` instead of `new-feature`. 😃 
- [ ] Let's try to use imperative mood while writing commit messages. For example, rather than writing `Readme` you write `Add Readme` similarly you should write `Add html and css` instead of just `html and css` 😃 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (KebebewAbabu) [reply .gitignore:9]: Hello Sumairq, 
Thank you for your nice review, I accept your comment and suggestions. Can you check it now?

**Comment by KebebewAbabu** [comment]
# Add html and CSS for hello world project
  
- [ ] Bug fix (unexpected generic font-family).
- [ ] New feature ( color;)
  - [ ] Run local stylelint config command(npx stylelint "**/*.{css,scss}" --fix  after changing the font.
# Checklist
 [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
```
Comment (Ginohmk) [inline .github/workflows/linters.yml:10]: - [Optional] On the PR description, it is advisable to write a short summary of what the PR is about, as in your case it is currently blank.
Screenshot from 2022-06-29 08-39-47

**Review by Ginohmk — APPROVED** [review]
Hi @KebebewAbabu , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**